x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -266,109 +249,76 @@ describe.skip('AllCasesListGeneric', () => {
   });
 
   it('renders the title column', async () => {
-    const res = appMockRenderer.render(<AllCasesList />);
+    appMockRenderer.render(<AllCasesList />);
 
-    expect(res.getByTestId('tableHeaderCell_title_0')).toBeInTheDocument();
+    expect(screen.getByTestId('tableHeaderCell_title_0')).toBeInTheDocument();
   });
 
   it('renders the updated on column', async () => {
-    const res = appMockRenderer.render(<AllCasesList />);
+    appMockRenderer.render(<AllCasesList />);
 
-    expect(res.getByTestId('tableHeaderCell_updatedAt_5')).toBeInTheDocument();
+    expect(screen.getByTestId('tableHeaderCell_updatedAt_5')).toBeInTheDocument();
   });
 
   it('renders the status column', async () => {
-    const res = appMockRenderer.render(<AllCasesList />);
+    appMockRenderer.render(<AllCasesList />);
 
-    expect(res.getByTestId('tableHeaderCell_status_7')).toBeInTheDocument();
+    expect(screen.getByTestId('tableHeaderCell_status_7')).toBeInTheDocument();
   });
 
   it('renders the severity column', async () => {
-    const res = appMockRenderer.render(<AllCasesList />);
+    appMockRenderer.render(<AllCasesList />);
 
-    expect(res.getByTestId('tableHeaderCell_severity_8')).toBeInTheDocument();
-  });
-
-  it('should render the case stats', () => {
-    const wrapper = mount(
-      <TestProviders>
-        <AllCasesList />
-      </TestProviders>
-    );
-    expect(wrapper.find('[data-test-subj="cases-count-stats"]')).toBeTruthy();
```

Comment:
`data-test-subj="cases-count-stats"` doesn't exist anywhere in `all_cases_list` and it's child components.

Reply:
<img src="https://media1.giphy.com/media/3zDdFSPALuCe6C43nM/giphy.gif"/>